js/setup.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const WIZARD_COUNT = 4;
+
+const options = {
+  firstNames: ['Иван', 'Хуан Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'],
+  lastNames: ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'],
+  coatColors: ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'],
+  eyesColors: ['black', 'red', 'blue', 'yellow', 'green']
+};
+
+const userDialog = document.querySelector('.setup');
+const setupSimilarList = document.querySelector('.setup-similar-list');
+const similarWizardTemplate = document.querySelector('#similar-wizard-template').content;
+
+const showUserDialog = () => userDialog.classList.remove('hidden');
+
+const showCharacters = () => userDialog.querySelector('.setup-similar').classList.remove('hidden');
+
+const getRandomArrValue = (arr) => arr[Math.floor(Math.random() * arr.length)];
+
+const getWizard = () => {
+  const wizard = {
+    name: getRandomArrValue(options.firstNames) + ' ' + getRandomArrValue(options.lastNames),
+    coatColor: getRandomArrValue(options.coatColors),
+    eyesColor: getRandomArrValue(options.eyesColors)
+  };
+
+  return wizard;
+};
```

Comment:
а что мешает сразу вернуть объект?) 
```suggestion
const getWizard = () => ( {
    name: getRandomArrValue(options.firstNames) + ' ' + getRandomArrValue(options.lastNames),
    coatColor: getRandomArrValue(options.coatColors),
    eyesColor: getRandomArrValue(options.eyesColors)
  });
```